flink-python/src/main/java/org/apache/flink/client/python/PythonDriver.java

```diff
@@ -126,4 +119,18 @@ static GatewayServer startGatewayServer() {
 		commands.addAll(pythonDriverOptions.getProgramArgs());
 		return commands;
 	}
+
+	/**
+	 * Constructs the Python configuration which will be used to prepare the environment of python process.
+	 *
+	 * @param pythonDriverOptions parsed Python command options
+	 */
+	static ReadableConfig constructPythonConfiguration(final PythonDriverOptions pythonDriverOptions) {
+		Configuration config = new Configuration();
+		config.set(
```

Comment:
Could we add some description on why only PYTHON_FILES is used to construct the configuration?